doc/level1.rst

```diff
@@ -1,26 +1,26 @@
 Autocrypt Level 1: Enabling encryption, avoiding annoyances
 ===========================================================
 
-This document describes the basic capabilities required for a mail app
-to be Autocrypt-capable at Level 1. The design of Level 1 is driven by
-usability concerns and by the realities of incremental deployment. A
-user may use both Autocrypt-enabled mail apps and traditional plain
-ones and we'd like to avoid annoyances like unexpected unreadable
-mails while supporting users who want to explicitly turn on
-encryption.
-
-For ease of implementation and deployment, Level 1 does not support
-multi-device synchronisation.  We intend to :doc:`support the multi-device
-use case (and other features) as part of Level 2<next-steps>`.  We
-want to keep Level 1 minimal enough that it's easy for developers to
-adopt it and we can start to drive efforts from real-life experiences
-as soon as possible.
+Autocrypt makes it easy for people to encrypt email.  This document
+describes the basic capabilities required for a mail app to be
+Autocrypt-capable at Level 1.
+
+The design of Level 1 is driven by usability concerns and by the
+realities of incremental deployment. A user may use both
+Autocrypt-enabled mail apps and traditional plain ones and we'd like to
+avoid annoyances like unexpected unreadable mails while supporting users
+who want to explicitly turn on encryption.
+
+For ease of implementation and deployment, Level 1 only contemplates the
+use of Autocrypt on a single device.  We intend to :doc:`support
```

Comment:
it's not "only contemplates" because we also have setup-message key sharing but "Level 1 focus on single-device use" or something.